content/reference/templates/_index.md

```diff
@@ -146,14 +146,14 @@ Similarly, provided a channel `$channel`, `$channel.Name` gives the name of the
 |.Channel.NSFW| Outputs whether this channel is NSFW or not.|
 |.Channel.OwnerID| The ID of the creator of threads as _int64_. Returns `0` for normal channels.|
 |.Channel.ParentID| The ID of the channel's parent (category), returns 0 if none.|
-|.Channel.PermissionOverwrites| A slice of [permission overwrite](https://discord.com/developers/docs/resources/channel#overwrite-object) structures applicable to the channel.|
+|.Channel.PermissionOverwrites| A slice of [Discord permission overwrite](https://discord.com/developers/docs/resources/channel#overwrite-object) structures applicable to the channel.|
 |.Channel.Position| Channel position from top-down.|
 |.Channel.Topic| The topic of the channel.|
-|.Channel.Type| The type of the channel. [Explained here.](https://discord.com/developers/docs/resources/channel#channel-object-channel-types)|
+|.Channel.Type| The type of the channel. Explained further in [Discord channel documentation](https://discord.com/developers/docs/resources/channel#channel-object-channel-types)|
```

Comment:
```suggestion
|.Channel.Type| The type of the channel. Explained further in [Discord's channel documentation](https://discord.com/developers/docs/resources/channel#channel-object-channel-types)|
```
Same reasoning as before. There may be / are more instances of this, but I don't want to flood this conversation with noise over an `'s`. Please go over similar links scoped to this PR and try to determine whether it makes more sense to have them in a possessive form, or to keep them as-is.